legacy/toml-config/src/main/java/io/zeebe/legacy/tomlconfig/LegacyConfigurationSupport.java

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.legacy.tomlconfig;
+
+import io.zeebe.legacy.tomlconfig.util.Loggers;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Function;
+
+public class LegacyConfigurationSupport {
+  private static final Map<String, Replacement> MAPPING_GATEWAY = new HashMap<>();
+  private static final Map<String, Function<String, String>> VALUE_CONVERTERS_GATEWAY =
+      new HashMap<>();
+  private static final Map<String, Replacement> MAPPING_BROKER = new HashMap<>();
+  private static final Map<String, Function<String, String>> VALUE_CONVERTERS_BROKER =
```

Comment:
Same as above, but more importantly, is this used anywhere? It seems in `checkForLegacyEnvironmentVariables` we use an empty map as mapping?

Reply:
Oops, done